src/main/java/com/paneedah/mwc/weapons/AuxiliaryAttachments.java

```diff
@@ -20,7 +20,7 @@ public class AuxiliaryAttachments {
     public static ItemAttachment<Weapon> BeowulfAction;
     public static ItemAttachment<Weapon> S710TricunActionPully;
     public static ItemAttachment<Weapon> S710TricunActionEjector;
-    public static ItemAttachment<Weapon> SA80Action;
+//    public static ItemAttachment<Weapon> SA80Action;
```

Comment:
It is preferable to just delete this, anyway thanks to GitHub if we ever need it we can just see it in the history.